dts/arm/cypress/psoc6_cm0.dtsi

```diff
@@ -15,6 +15,306 @@
 
 		/delete-node/ cpu@1;
 	};
+
+	soc {
+		intmux: intmux@40210020 {
+			compatible = "cypress,psoc6-intmux";
+			reg = <0x40210020 0x20>;
+			label = "INTMUX";
+			status = "disabled";
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			intmux_ch0: interrupt-controller@0 {
+				compatible = "cypress,psoc6-intmux-ch";
+				reg = <0x0>;
+				#interrupt-cells = <2>;
+				interrupt-controller;
+				interrupts = <0 0>;
```

Comment:
It's your decision, but do you think you might want to make the default priority a bit lower (assuming 0 means "highest priority") for these?

We did that recently for Nordic SoCs to make it easier for users to increase the priority of just some interrupts without having to change priorities for other unrelated nodes.

Reply:
fully agreed, thanks!